nimare/decode/continuous.py

```diff
@@ -220,6 +234,19 @@ def transform(self, img):
 class CorrelationDistributionDecoder(Decoder):
     """Decode an unthresholded image by correlating the image with study-wise images.
 
+    .. versionadded:: 0.0.1
+
+    .. versionchanged:: 0.0.2
+        [ENH] Change the order of the first two parameters (img, dataset) -> (dataset, img)
+
+    .. versionchanged:: 0.0.3
+        [ENH] New parameter *feature_group=None*.
+        [ENH] Convert decoder :func:`corr_dist_decode` to class
+        :class:`CorrelationDistributionDecoder`.
```

Comment:
It looks like newlines aren't respected, so this renders as a paragraph.

Reply:
We could also use bullet points?
![Screen Shot 2021-04-30 at 12 18 56 PM](https://user-images.githubusercontent.com/52050407/116724412-9f7b6580-a9ae-11eb-93ba-8b76f2d52b72.png)

Reply:
That does look much better!